src/api/database/models/OngoingMeetingCount.ts

```diff
@@ -19,6 +19,9 @@ class OngoingMeetingCount extends Model {
   @Column(STRING)
   TMAdminUserId: string;
```

Comment:
We don't need this column any more. It's confirmed that all meetings can share the same admin user id.